index.html

```diff
@@ -159,7 +165,7 @@ <h5>Skills</h5>
          </ul>
       </div>
    </section>
-   <section class="contact">
+   <section id="contact">
```

Comment:
- Please align the button `Get in touch`  to the left as in the Figma design, currently yours is centered. 
- [Opotional] Some background images are missing, I suggest you include them so as it gives more UX to your design .
 ##### Your design
![Screenshot from 2022-06-10 09-01-41](https://user-images.githubusercontent.com/58771507/173020315-b628bd29-0bbe-4e5e-b691-1ea91be798b8.png)
 
##### Figma design
![Screenshot from 2022-06-10 09-05-14](https://user-images.githubusercontent.com/58771507/173020510-3955d4cd-f633-4faf-b357-1bf961f3d34f.png)